### ISSUE TITLE
Docker command line options

### ISSUE THREAD

**Issue by MasterSmurf** [zotify-dev/zotify#126]
Got docker running like described in the README.md. First the command like ask for `Username` and `Password:`. Then `Search:` and `ID:`. Everything works like intended.

But how can you use the command line options with `docker run`, like changing the format or download a specific playlist URL?

**Comment by MasterSmurf** [comment]
To specify, I run the docker container with
`docker run --rm -v "$PWD/Zotify Music:/root/Music/Zotify Music" -v "$PWD/Zotify Podcasts:/root/Music/Zotify Podcasts" -it zotify`
How to use the non-predefined command lines?

**Comment by jacobslusser** [comment]
The way the `Dockerfile` is written, I believe you would actually need to "bake-in" the additional arguments to the Dockerfile and then rebuild. Rather than muck with that, the approach I take is to simply open an interactive shell in the docker container like so:
```
docker run --rm -v "$PWD/Zotify Music:/root/Music/Zotify Music" -v "$PWD/Zotify Podcasts:/root/Music/Zotify Podcasts" -it zotify /bin/sh
```

At this point, you are running in the container and can invoke Zotify like you had installed it with Python:
```bash
python3 -m zotify --help
```

**Comment by MasterSmurf** [comment]
Yeah, just launching the container shell and running zotify with python, makes sense now. Anyway, thank you for the answer, made my day👍
